Example/ReusableDemo/TableViewController.swift

```diff
@@ -27,7 +27,8 @@ final class TableViewController: UITableViewController {
   }
 
   override func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
-    let view = MyHeaderTableView.loadFromNib()
+    let view = MyHeaderTableView(frame: CGRect(x: 0, y: 0, width: tableView.bounds.size.width, height: self.tableView(tableView, heightForHeaderInSection: section)))
+    view.loadNibContent()
```

Comment:
Could be the occasion to:

* Break that line 30 in 2 lines (`let frame = CGRect(…)` + `let view = MyHeaderTableView(frame: frame)`
* Add a comment just above this call to `init` to remind the reader of the example project that "this calls the overriden init of `MyHeaderTableView` which loads its content from its Nib automatically via `loadNibContent()`" or something?